ArduPlane/Log.cpp

```diff
@@ -505,7 +509,7 @@ void Plane::Log_Read(uint16_t list_entry, int16_t start_page, int16_t end_page)
 
     cliSerial->printf("%s\n", HAL_BOARD_NAME);
 
-	DataFlash.LogReadProcess(list_entry, start_page, end_page,
+   DataFlash.LogReadProcess(list_entry, start_page, end_page,
```

Comment:
Something must have gone wrong with some of the spacing-related reversals... Frankly, I don't know an easier way of fixing this than doing what I did initially, i.e., just fixing the formatting of the files in question. But then formatting changes are supposed to be separate... 

Re:rebasing, I'd appreciate if you did this and told me how, so that I can do it myself next time :)